### PR TITLE
Updated Dolt to version 0.26.11

### PR DESCRIPTION
**Pull request by tiksn**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20482)

### CODE REVIEW

File: manifests/d/DoltHub/Dolt/0.26.11/DoltHub.Dolt.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+PackageIdentifier: DoltHub.Dolt
+Publisher: DoltHub
```
Comment (ItzLevvie) [inline manifests/d/DoltHub/Dolt/0.26.11/DoltHub.Dolt.yaml:2]:
```suggestion
Publisher: Liquidata
```

File: manifests/d/DoltHub/Dolt/0.26.11/DoltHub.Dolt.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+- Versioning
+- git-for-data
+ShortDescription: DoltHub.Dolt
+Description: Dolt is a SQL database that you can fork, clone, branch, merge, push and pull just like a git repository. Connect to Dolt just like any MySQL database to run queries or update the data using SQL commands. Use the command line interface to import CSV files, commit your changes, push them to a remote, or merge your teammate's changes.
+PackageUrl: https://www.dolthub.com/
+Installers:
+- Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://github.com/dolthub/dolt/releases/download/v0.26.11/dolt-windows-amd64.msi
+  InstallerSha256: 462BE4C27D8E77A4397BE8A938137E0AFEB8C235EE219A8278BB9C176D37ECE5
```
Comment (ItzLevvie) [inline manifests/d/DoltHub/Dolt/0.26.11/DoltHub.Dolt.yaml:21]:
```suggestion
  InstallerSha256: 462BE4C27D8E77A4397BE8A938137E0AFEB8C235EE219A8278BB9C176D37ECE5
  ProductCode: "{7EA03F03-5FE7-4DE4-9D80-6AD81E459A6D}"
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/d/DoltHub/Dolt/0.26.11/DoltHub.Dolt.yaml (hunk 1 above)
Comment (tiksn) [reply manifests/d/DoltHub/Dolt/0.26.11/DoltHub.Dolt.yaml:2]: Company is renamed, now it is called  DoltHub, Inc.

Comment (OfficialEsco) [reply manifests/d/DoltHub/Dolt/0.26.11/DoltHub.Dolt.yaml:2]: Dosn't matter when the installer shows the `Publisher` as `Liquidata`

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/20482#discussion_r667548269

**Pull request closed without merging**